catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -24,6 +24,13 @@
         <AD name="Enable Notifications" id="notificationEnabled" required="false" type="Boolean"
             default="true"
             description="Check to enable notifications."/>
+
+    </OCD>
+
+    <OCD name="Historian" id="ddf.catalog.history.Historian">
+        <AD name="Enable Versioning" id="historyEnabled" required="true" type="Boolean"
+            default="false"
```

Comment:
I thought someone told me to make it false by default (was previously true.) If we want to I an flip the switch now.